src/pages/user-likes.js

```diff
@@ -36,15 +39,27 @@ class UserLikesPage extends React.Component {
   static async fetchData(props, client) {
     try {
       let userInfo = client.userInfo(props.params.username);
-
       getStore().dispatch(addUser(await userInfo));
+
+      let likedPosts = await client.getLikedPosts(props.params.username);
+
+      let liked_posts_len = likedPosts.length;
+      for (let i = 0; i < liked_posts_len; i++) {
+        getStore().dispatch(addUser(likedPosts[i].user));
+      }
+
+      getStore().dispatch(setUserPosts(props.current_user.id, likedPosts));
```

Comment:
I just realized this is wrong, should be `setPostsToRiver`, this way has unexpected behaviors when going back and forth from Posts to Likes.